Graphy.py

```diff
@@ -287,103 +307,87 @@ def GUI(DAILY_MEAN_2019 = None, DAILY_MEAN_2020 = None, WEEKLY_MEDIAN_2019 = Non
-                        month = 6
-                    elif selectedMonth == ['October']:
-                        month = 7
-                    elif selectedMonth == ['November']:
-                        month = 8
-                    elif selectedMonth == ['December']:
-                        month = 9
-                ## TRADITIONAL MONTH ORDERING (ie, JAN 0, FEB = 2 etc)
-                elif location == 'External': #ie, ensuring that JAN == 1 - NOT INLCUDED FOR NOW, PLACEHOLDER FOR NEW DATA #BROKEN - FIX LATER - CURRENTLY EXCLUDED
-                    if selectedMonth == ['January']:
-                        month = 0    
-                    elif selectedMonth == ['February']:
+                
+                if selectedMonth == ['January']:
+                    month = 0    
+                elif selectedMonth == ['February']:
                         month = 1
-                    elif selectedMonth == ['March']:
-                        month = 2
-                    elif selectedMonth == ['April']:
-                        month = 3
-                    elif selectedMonth == ['May']:
-                        month = 4
-                    elif selectedMonth == ['June']:
-                        month = 5
-                    elif selectedMonth == ['July']:
-                        month = 6
-                    elif selectedMonth == ['August']:
-                        month = 7
-                    elif selectedMonth == ['September']:
-                        month = 8
-                    elif selectedMonth == ['October']:
-                        month = 9
-                    elif selectedMonth == ['November']:
-                        month = 10
-                    elif selectedMonth == ['December']:
-                        month = 11
+                elif selectedMonth == ['March']:
+                    month = 2
+                elif selectedMonth == ['April']:
+                    month = 3
+                elif selectedMonth == ['May']:
+                    month = 4
+                elif selectedMonth == ['June']:
+                    month = 5
+                elif selectedMonth == ['July']:
+                    month = 6
+                elif selectedMonth == ['August']:
+                    month = 7
+                elif selectedMonth == ['September']:
+                    month = 8
+                elif selectedMonth == ['October']:
+                    month = 9
+                elif selectedMonth == ['November']:
+                    month = 10
+                elif selectedMonth == ['December']:
+                    month = 11
+            ## TRADITIONAL MONTH ORDERING (ie, JAN 0, FEB = 2 etc)
             else: 
                 print('please select a month')
             #do yearly logic
             if values['Year']:  #determine the year
                 selectedYear = values['Year'] 
                 #do year logic
-                if selectedYear == ['2019']:
+                if selectedYear == ['2018']:
```

Comment:
why not just convert from `"2019"` -> int `2019` look up string to number conversion. Dont need to so this janky if statement block

Reply:
It was just how the GUI returns the selected year.